iset.mm

```diff
@@ -62575,6 +62585,20 @@ the cancellation property (fifth hypothesis), show that the relation
       CVTYIXPABCWAWBWCWDVR $.
   $}
 
+  $( Excluded middle is equivalent to the power set of a singleton having two
```

Comment:
"the power set of `1o`"

Reply:
I suspect it doesn't matter whether you are looking at `1o` or a different singleton, but I'll take your suggested wording. There are cases where the comment should describe the result in terms of what it means rather than just repeating the formal theorem directly, but to say "a singleton" is, I suppose, a bit too far beyond what we actually prove.

Reply:
It is a bit far indeed. When formalizing mathematics, the risk of errors moves from the proof itself (which is certain if we trust the kernel), to the interpretation of the formal statement. This is not purely theoretical, but is a real concern (I read a particularly acute version of this stance in a survey by Wiedijk). That's why I'm often careful about comments being precise enough.